src/Framework/NativeMethods.cs

```diff
@@ -0,0 +1,1695 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
```

Comment:
Nit:
Extra //

Is this just a copy of NativeMethodsShared but not in Shared? Why do some later changes refer to Tasks.NativeMethods?

Reply:
I hadn't completed the move--out of

1. Copy
2. Delete old

I had done only one!

The `Tasks.NativeMethods` change is a result of newly-created ambiguity between this new `Microsoft.Build.Framework.NativeMethods` and `Microsoft.Build.Tasks.NativeMethods`. The fix was to qualify the reference to not be ambiguous any more.